db/seeds.rb

```diff
@@ -10,13 +10,13 @@ def initialize
   end
 
   def generate_categories
-    categories = Category.create([{ name: "Main meals"}, { name: "Pizzas" },
+    @categories = Category.create([{ name: "Main meals"}, { name: "Pizzas" },
                               { name: "Drinks" },    { name: "Salads"},
-                              { name: "Desserts" },  { name: "Vegetarian Items" }
+                              { name: "Desserts" },  { name: "Vegetarian Items" }])
```

Comment:
Line is too long. [83/80]<br>Align the elements of an array literal if they span more than one line.